frontend/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,16 @@
+import { Typography, Box } from '@material-ui/core';
+import useStyles from './styles'
+
+
+const Footer = ({text=`Copyright \u00a9 TourDeFoos 2021`}) => {
```

Comment:
Hej, dodaj tutaj rok, który będzie się aktualizował.
Możesz wyciągnąć go z "new Data()".
Będzie fajny bajer, którym zabłyśnie ten komponent :)

Reply:
A chcemy, żeby rok sie aktualizował? Ja aż do dzisiaj myślałam, że ten rok to po prostu data utworzenia apki :)